bundle/edu.gemini.spModel.pio/src/main/scala/edu/gemini/spModel/pio/codec/ParamCodec.scala

```diff
@@ -1,5 +1,8 @@
 package edu.gemini.spModel.pio.codec
 
+import edu.gemini.spModel.core.Deflated
+import sun.misc.{BASE64Decoder, BASE64Encoder}
```

Comment:
In Java 8 there are Base64 Encoder/Decoders:
https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html